### PR TITLE
Shotgun csv import export

### PR DESCRIPTION
**Pull request by BadSingleton**
Adds a simple C# demo on how to import and export a timeline of a sequence into Unity from Shotgun

### CODE REVIEW

File: ShotgunImportExport/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+--------
+* Create a new project
+* Import the [package](Bin/ShotgunImportExport.unitypackage) into your project
+* Then through the menu `Shotgun CSV > Import` import the example csv file
+    * Alternatively, change the `pathToFile` and `filePath` fields to the file you want to use
+* Make edits to the timeline
+* Then through the menu `Shotgun CSV > Export` export to Shotgun-compatible tab-separated values. This will create a file name `export_from_unity.csv` in the same directory as the imported CSV file
+* Use the import function of Shotgun to import the edits into your sequence
+
+##Limitations
```
Comment (mfe) [inline ShotgunImportExport/README.md]: Detail but: github does not interpret well the `##`, so I would use `-------- `.

File: ShotgunImportExport/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+==========================
+
+This is a simplified example of how to create a timeline in Unity using a shot sequence from Shotgun.
+
+How to use
+--------
+* Create a new project
+* Import the [package](Bin/ShotgunImportExport.unitypackage) into your project
+* Then through the menu `Shotgun CSV > Import` import the example csv file
+    * Alternatively, change the `pathToFile` and `filePath` fields to the file you want to use
```
Comment (mfe) [inline ShotgunImportExport/README.md]: `fileName` instead of `filePath` ?

**Comment by mfe** [comment]
Suggestions : 
* Timeline Asset needs to be saved, right now there is no playable saved in the Asset folder
* If no binding is done in the script, creating the scene Game Object with a playable director component does not bring value. So it would be enough to just save the Timeline Asset.
* Move the menu in `Window > General > Film/TV toolbox > Shotgun`, name the entry `Import CSV` / `Export CSV`.

If you have questions about timeline, @vladal (in slack) / @clusty is our expert :)

File: ShotgunImportExport/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+This is a simplified API example of how to create a timeline in Unity using a shot sequence from Shotgun.
+
+How to use
+--------
+* Create a new project
+* Import the [package](Bin/ShotgunImportExport.unitypackage) into your project
+* Then through the menu `Window > General > Film/TV toolbox > Shotgun > Import CSV` import the example csv file
+    * Alternatively, change the `k_pathToFile` and `k_importFileName` fields to the file you want to use
+* Make edits to the timeline asset
+* Then through the menu `Window > General > Film/TV toolbox > Shotgun > Import CSV` export to Shotgun-compatible tab-separated values. This will create a file name `export_from_unity.csv` in the same directory as the imported CSV file
```
Comment (clusty) [inline ShotgunImportExport/README.md]: Sorry to do this to you, would you mind changed the menu entries to Window/Film-TV toolbox/Shotgun/Import CSV

**Comment by clusty** [comment]
I noticed that the activation clips generated leave gaps: First one ends at 5:41, while the second starts at 5:45 . Is this done on purpose ?

Was wondering what is a "Shot". Is it a timeline ?

File: ShotgunImportExport/README.md (hunk 3 above)
Comment (BadSingleton) [reply ShotgunImportExport/README.md]: I don't mind

**Comment by mfe** [comment]
(the package is not up to date)

**Comment by clusty** [comment]
All good.

**Review by clusty — APPROVED** [review]

**Review by mfe — APPROVED** [review]
LGTM, will merge and then integrate it as a sample.
Thanks !

**Pull request merged**